notebooks/metrics.py_API_Documentation.ipynb

```diff
@@ -190,8 +190,9 @@
     "\n",
     "APE is based on the absolute relative pose between two poses $P_{ref,i}, P_{est,i} \\in \\mathrm{SE}(3)$ at timestamp $i$:\n",
     "\\begin{equation*}\n",
-    "E_i = P_{ref,i} \\ominus P_{est,i} = P_{ref,i}^{-1} P_{est,i} \\in \\mathrm{SE}(3)\n",
+    "E_i = P_{est,i} \\ominus P_{ref,i} = P_{ref,i}^{-1} P_{est,i} \\in \\mathrm{SE}(3)\n",
     "\\end{equation*}\n",
+    "where $\\ominus$ is the inverse compositional operator [Kummerle et al. 2009].\n",
```

Comment:
Kümmerle is an important reference for this notebook, but AFAIK the "ominus" operator was first used in this context in [Lu-1997] (http://www-robotics.usc.edu/~gaurav/CS547/milios_map.pdf , p15)

Following that paper, I'd rephrase it as: *[...] is the inverse pose compounding operator, which takes two poses and gives the relative pose [Lu-1997].*

Reply:
That's right!